mamba/mamba/utils.py

```diff
@@ -274,6 +277,7 @@ def get_base_url(url, name=None):
     api_ctx.retry_backoff = context.remote_backoff_factor
     api_ctx.add_pip_as_python_dependency = context.add_pip_as_python_dependency
     api_ctx.use_only_tar_bz2 = context.use_only_tar_bz2
+    api_ctx.repodata_use_zst = True
```

Comment:
I hard-coded all around to true, and I should obviously pass it along from the context.

What's your preference on the path forwards? Should i make `repodata_use_zst` configurable using an env var (a la `MAMBA_REPODATA_USE_ZST`), read that in here and set it accordingly?